src/main/java/org/craftercms/studio/impl/v2/service/policy/validators/ContentTypePolicyValidator.java

```diff
@@ -60,4 +59,8 @@ public void validate(HierarchicalConfiguration<?> config, Action action) throws
         }
     }
 
+    @Override
+    public void validate(HierarchicalConfiguration<?> permittedConfig, HierarchicalConfiguration<?> deniedConfig, Action action) throws ValidationException {
```

Comment:
@phuongnq it would be better to implement the `denied` for all existing validators right away, not just for the mime-type

Reply:
you are right, it only makes sense for the validators that perform a comparison against a list, please go ahead and add it for the content-type and then we can document the ones that do support it